pybamm/pybammdata.py

```diff
@@ -0,0 +1,48 @@
+import pooch
+
+
+class DataLoader:
+    def __init__(self):
+        self.path = pooch.os_cache("pybamm")
+        self.odie = pooch.create(
+            path=self.path,
+            base_url="https://github.com/pybamm-team/pybamm-data/releases/download/v1.0.0",
+            registry={
```

Comment:
Could you add a comment that the data under `pybamm/input/parameters/` has not been included at this time?

Reply:
added this to the PR description